lib/hanrahashiv.py

```diff
@@ -0,0 +1,84 @@
+from manim import Tex, MathTex
+from manim import UP, DOWN, LEFT, DL
+from manim import VGroup
+from manim import Scene
+from manim import ReplacementTransform
+
+class FormulaModifications(Scene):
+
+# FIXME մարդավարի docstring գրել ամեն ֆունկցիայի համար
+#  միգուցե էս ամենը տեղափոխել QarakusiScene (որտև մենակ միանդամների մեջ չի, որ պետք կգան)
+    """
+        new_sequence addresses each of formula's item to it's new position, including '$\cdot$', ' + '
+        to work properly, formula must be written in such a way that every item is in different apostrophes like so
+        formula = Tex('a', '$\cdot$', 'b')
+    """
+
+    def rearrange_formula(self, formula : Tex or MathTex, new_sequence : list, move_up : list, move_down : list):
```

Comment:
Did you know, that `Tex` is a subclass of `MathTex`?

Reply:
Is the `new_sequence` a permutation of `range(len(tex_string_list))`

Reply:
հա, այսինքն կարայի formula:MathTex գրել ու վսյո՞

հա, վերադասավորություն ա

Reply:
հա, կարայիր, ու լրիվ ճիշտ կէղներ